### DISCUSSION TITLE
Version 3.0 ETA

### DISCUSSION THREAD

**Discussion by ghost** [retypeapp/retype#478] [General]
Hi, 
Really like Retype as a SSG, just wondered if there is an ETA on v3.0 as the only thing holding us back is mermaid functionality available in v9.* 
Thanks D

**Comment by geoffreymcgill** [comment]
With some luck, a `v3.0.0-preview` release should be available next week. The final `v3.0.0` release will be available shortly after the `-preview`.

Retype v3.0.0 includes Mermaid v9.3.0.
